changelogs/fragments/port_exception.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Handles exception when translating an unknown port to its service name.
```

Comment:
```suggestion
  - eos_acls - Handles exceptions when translating an unknown port number to its service name. The command would default to using the port number instead of the service name.
```